torch/random.py

```diff
@@ -65,9 +62,6 @@ def seed() -> int:
     if not torch.mps._is_in_bad_fork():
         torch.mps.manual_seed(seed)
 
-    if hasattr(torch, 'xpu') and not torch.xpu._is_in_bad_fork():
-        torch.xpu.manual_seed_all(seed)
```

Comment:
We will add these back in next PR, right?

Reply:
yes, this code will be back in `Generator`'s PR. If we don't remove this code in this PR, it will raise an error in line 68.